bzt/bza.py

```diff
@@ -750,7 +752,12 @@ def get_addr(self):
 
         self._request(self.address + '/api/latest/proxy/recording/clear', method='POST')
 
-        return 'http://%s:%s' % (proxy_info['host'], proxy_info['port'])
+        return '%s:%s@%s:%s' % (
```

Comment:
is losing `http://` intentional? This is surfaced by unit test...

Reply:
yes, if you don't specify protocol it will do it for both